src/AnalyzerBot/utils/commentUtil.ts

```diff
@@ -2,14 +2,14 @@ import { LogSeverity, LogType } from '../../Parser';
 import { Comment, CommentStructure } from '../@types/CommentTypes';
 import { MessageUtil } from './message.util';
 
-export function groupComments(logs: LogType[], suppressRules: Set<string>): Comment[] {
+export function groupComments(logs: LogType[], suppressRules?: Array<string>): Comment[] {
```

Comment:
`suppressRules` from config is not nullable, `[]` is the default value set from the yargs options. i think we don't need to type this param as nullable.